cartography/intel/aws/iam.py

```diff
@@ -224,6 +224,18 @@ def load_groups(neo4j_session, groups, current_aws_account_id, aws_update_tag):
         )
 
 
+@timeit
+def _parse_principal_entries(principal):
```

Comment:
Please add a comment to this function describing the shape of the data returned e.g.
`List of tuples of the form (principal_type, principal_value) e.g. ('AWS', 'role-name')`